g2p_registry_rest_api/models/group.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
+
+import pydantic
+
+from .group_membership import GroupMembersInfoIn  # fmt: skip
+from .group_membership import GroupMembersInfoOut  # fmt: skip
+from .registrant import RegistrantInfoIn  # fmt: skip
+from .registrant import RegistrantInfoOut  # fmt: skip
+from .registrant import Relationship1In  # fmt: skip
+from .registrant import Relationship1Out  # fmt: skip
+from .registrant import Relationship2In  # fmt: skip
+from .registrant import Relationship2Out  # fmt: skip
```

Comment:
Why do we need to skip format checks? Wasn't the change in isort config done to fix this issue?

Reply:
The fix on the config doesn't fix the problem in the isort and black that's why I did it this way.